scripts/release/utils.injector.js

```diff
@@ -0,0 +1,274 @@
+    const data = dataClass.children.find((item) => item.id === id);
+    if (!data?.flags?.isPublic) {
+      continue;
+    }
+
+    for (const signature of data.signatures) {
+      result.push({ h3: signature.name });
+      result.push({ p: json2mdTrim(getComment(signature)) });
+      if (signature.parameters) {
+        const parameters = signature.parameters?.map((item) => {
+          return {
+            name: item.name,
+            type: item.type.name,
+            description: getComment(item)
+          };
+        });
+        result.push(...generateParameter(parameters));
+      }
+      result.push(
+        ...generateReturn({
+          // Typedoc generates 2 id for type. First is declaration for wrapper and Second is signature for info.
+          // The signature contains inside the declaration.
+          type: mappedSignatures.find((item) => item.id - 1 === id).returnType,
+          description: getReturnComment(signature)
+        })
+      );
+    }
+  }
+  return result;
+};
+
+/**
+ * Generate and return Class content in structure of json2md.
+ * @param {json} json json
+ * @param {string} title title for header level 1.
+ * @returns {array} json2md array
+ */
+const generateClassDocument = (json, title) => {
+  const result = [];
+  const dataClassesIDs = json.groups.find((item) => item?.title === 'Classes')?.children;
+
+  if (dataClassesIDs?.length <= 0) {
+    error("Can't find Class.");
+    return result;
+  }
+
+  const mappedSignatures = json.mappedSignatures;
+
+  for (const classID of dataClassesIDs) {
+    const dataClass = json.children.find((item) => item.id === classID);
+    if (!dataClass) {
+      continue;
+    }
+    result.push({ h1: title || dataClass.name });
+
+    const dataConstructorIDs = dataClass.groups.find((item) => item.title === 'Constructors')?.children;
+    const dataMethodIDs = dataClass.groups.find((item) => item.title === 'Accessors')?.children;
+    const dataFunctionIDs = dataClass.groups.find((item) => item.title === 'Methods')?.children;
+
+    dataConstructorIDs && result.push(...generateConstructor(dataConstructorIDs, dataClass));
+    dataMethodIDs && result.push(...generateAccessor(dataMethodIDs, dataClass, mappedSignatures));
+    dataMethodIDs && result.push(...generateMethod(dataFunctionIDs, dataClass, mappedSignatures));
+  }
+
+  return result;
+};
+
+/**
+ * Return trimmed string in Camel case to use for file name
+ * @param {string} text string
+ * @returns {string} Camel case string
+ */
+const toKebabCase = (text) => {
+  return text.trim().toLowerCase().replaceAll(' ', '-');
+};
+
+/**
+ * Generate md file from JSON
+ * @returns {void}
+ */
+const generateMD = () => {
+  for (const { entry, title, output } of generateDocList) {
+    // entry is input ts file path
+    const entryPoint = entry.replaceAll(ELEMENT_SOURCE, ELEMENT_DIST);
+    const { dir, name } = path.parse(entryPoint);
+    const inputFile = path.join(dir, name + '.json'); // convert .ts to .json
```

Comment:
Let's remove the comment and update `entry is input ts file path` to `build generated json file path from ts file path`
```suggestion
    const inputFile = path.join(dir, name + '.json');
```